GiEnJul/ClientApp/src/routes/registerGiver/ContactInfo.tsx

```diff
@@ -22,24 +24,64 @@ const ContactInfo: React.FC<Props> = ({ nextStep, prevStep, handlefullnameChange
         prevStep();
     }
 
+    const classes = useStyles();
+
     return (
-        <div>
-            <h1>Bli giver</h1>
-            <h3>Kontaktinformasjon</h3>
-            <label> Navn
-                <input type="text" onChange={handlefullnameChange} value={values.fullname} />
-            </label>
-            <label> Epost
-                <input type="text" onChange={handleEmailChange} value={values.email} />
-            </label>
-            <label> Mobilnummer
-                <input type="text" onChange={handleTlfChange} value={values.phoneNumber} />
-            </label>
-            <ButtonToolbar>
-                <Button onClick={Previous}>Tilbake</Button>
-                <Button onClick={Continue} >Neste</Button>
-            </ButtonToolbar>
-        </div>
+        <Container>
+            <FormControl 
+             variant="outlined"     
+             fullWidth
+             required
+             margin = "normal"
+            style={{width: '100%', marginTop: '5px'}}>
+                <TextField
+                    variant="outlined"
+                    margin="normal"
+                    required
+                    fullWidth
+                    id="fullname"
+                    label="Fult navn"
+                    name="fullname"
+                    autoComplete="name"
+                    autoFocus
+                    value={values.fullname}
+                    onChange={handlefullnameChange}
+                />
+                <TextField
+                    variant="outlined"
+                    margin="normal"
+                    required
+                    fullWidth
+                    id="email"
+                    label="Epost"
+                    name="email"
+                    autoComplete="email"
+                    autoFocus
+                    value={values.email}
+                    onChange={handleEmailChange} 
+                />
+                <TextField
+                    variant="outlined"
+                    margin="normal"
+                    required
+                    fullWidth
+                    name="phoneNumber"
+                    label="Telefonnummer"
+                    id="phoneNumber"
+                    autoComplete="tel"
+                    value={values.phoneNumber}
+                    onChange={handleTlfChange} 
+                />
```

Comment:
The `TextFields` here probably apply to all the `<input type="text">` fields on the page. Because of this I think the changes should be applied to the `InputValidator` micro component. This may be a rather big task to implement though: 
* The `InputValidator` is currently not supporting the `TextField` attributes.
* The `InputValidator` is also used by the `type=number` attribute, which could not be supported by the `TextField` component.
Because of these complications I would suggest to create an issue that refactors this.

Reply:
I agree, an issue on this is the best solution!